.gitignore

```diff
@@ -8,3 +8,7 @@
 
 ## System ##
 .DS_Store
+.eslintcache
+.vscode/
+node_modules/
```

Comment:
I believe that it would prove worthwhile to use the bootstrap's original `.gitignore` because it has more files excluded from being staged to a commit

Reply:
ill switch to the boostrap's .gitignore once i merge this PR, for the HTML CSS training i use the .gitignore i made myself, now that im working on angular ill use the bootstrap one